<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistWidget</class>
 <widget class="QWidget" name="RegistWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>395</width>
     <height>395</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>395</width>
     <height>395</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>395</width>
     <height>395</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>49</y>
       <width>53</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>用户名:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>115</y>
       <width>45</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>181</y>
       <width>68</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>确认密码:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>247</y>
       <width>45</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>性别：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonRegist">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>310</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>确认注册</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonBack">
     <property name="geometry">
      <rect>
       <x>244</x>
       <y>313</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSure">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>180</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPswd">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditUser">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxSex">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>250</y>
       <width>60</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>男</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>女</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>保密</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>152</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QLabel" name="labelqqId">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>53</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>130</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="ButtonSure">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>130</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>88</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
